core.md

```diff
@@ -57,23 +57,17 @@ You may [apply to join some teams](https://links.neoforged.net/apply), but some
 
 #### Steering Council
 
-The Steering Council is the overall management and oversight of the Project.
+The Steering Council are the representatives of the Project, final check for if a Project member is to be demoted or banned, and tiebreakers for perfectly split votes.
```

Comment:
```suggestion
The Steering Council is the representative of the Project in the community. It has the power to cast tiebreaking votes in Team votes with a perfectly split outcome, and it also renders a final decisions in cases involving the demotion or removal from the community of Project members. 
```
The use of is here is correct as it refers to the council as an institution, and not its members (see the original governance pr).

Reply:
closing comments as Steering Council was completely redone